### DISCUSSION TITLE
Scraping and parsing Goodreads user books data in Node.js | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#74] [Announcements]
# Scraping and parsing Goodreads user books data in Node.js | Leo's dev blog – stories, insights, and ideas

Scraping Goodreads books data from user's RSS feed and parse to JSON in Node.js

https://www.leohuynh.dev/snippets/fetch-goodreads-user-books-data

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
